scala/scala.bzl

```diff
@@ -89,7 +89,7 @@ def _build_nosrc_jar(ctx, buildijar):
     # this ensures the file is not empty
     resources += "META-INF/MANIFEST.MF=%s\n" % ctx.outputs.manifest.path
 
-    zipper_arg_path = ctx.actions.declare_file("%s_zipper_args" % ctx.outputs.jar.path)
+    zipper_arg_path = ctx.actions.declare_file("%s_zipper_args" % ctx.label.name)
```

Comment:
it is not directly, but discovered along the way of debugging. It is not a bug, it is just really junky to use the output path as the path name because it nests a totally giant path.